### PR TITLE
Added redirection to meshery catalog from meshmap page

### LINKED ISSUE #4098: Redirecting to meshery catalog from meshmap page
### Current Behavior
The meshmap page here https://layer5.io/cloud-native-management/meshmap/ contains an animation of floating catalog cards. 

image

### Desired Situation
By clicking on this animated graphic, the user should be redirected to the meshery catalog here: https://meshery.io/catalog

---
<h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
- 🎨 Wireframes and designs for Layer5 site in [Figma](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). 
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io).

Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

### PR DESCRIPTION
**Pull request by AkashS20**
**Description**

This PR fixes #4098 

https://user-images.githubusercontent.com/92908347/232225647-fc3ca120-95b2-46de-9b1c-0e1f3261cba4.mp4

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit dc934aaeb936351438e323e1d4091966cb5f2367 at: https://643aa70668a7fe258687255f--layer5.netlify.app

File: src/sections/Meshmap/meshmap-catalog.js [hunk 1]
```diff
@@ -440,60 +440,62 @@ const Catalog = () => {
...
-              <div className="slide thirteen">
-                <img src={Jwt_transformer} />
-              </div>
-              <div className="slide fourteen">
-                <img src={Multicluster} />
-              </div>
-              <div className="slide fifteen">
-                <img src={Http_metrics} />
+        <a href="https://meshery.io/catalog">
+          <div className="catalog-wrapper ">
```
Comment (Shivam-AfA) [inline src/sections/Meshmap/meshmap-catalog.js:444]: @AkashS20 Thanks for this. Although please ensure that unnecessary line changes are not present in the PR. It makes the PR cluttered and also hard to review.

Comment (AkashS20) [reply src/sections/Meshmap/meshmap-catalog.js:444]: Although I usually avoid it, in this instance, I had to add two spaces at the start of each line to maintain the indentation, which was then detected as changes by git.

Comment (Shivam-AfA) [reply src/sections/Meshmap/meshmap-catalog.js:444]: Gotcha 👍

**Pull request merged**